playwright/support/helper.ts

```diff
@@ -61,6 +61,15 @@ export const checkAccessibility = async (
     .disableRules([...preDisabledRules, ...disableRules])
     .analyze();
 
+  if (accessibilityScanResults.incomplete.length > 0) {
+    console.warn(
+      `\nACCESSIBILITY SCAN INCOMPLETE. Incomplete rules: ${accessibilityScanResults.incomplete.map(
```

Comment:
suggestion: I think it might be worth adding the component that the incomplete result has come from right now it looks like below so doesn't have much context.

![image](https://github.com/user-attachments/assets/5c17557c-e3c2-4021-9414-2921e3757f96)

Reply:
suggestion (non-blocking): It might also be worth adding a check for `process.env.CI` and only adding the warn locally rather than polluting the workflow runs. I think if we want this information gathering during CI runs we can generate a report or something but happy for that to be a longer term goal. Maybe we can just put a single small message in the CI saying the following components have incomplete accessibility checks or some sort of count